File: swea/D2/1986.지그재그 숫자/1986.cpp
#include <iostream>

using namespace std;

int T; 

int main(){
    cin >> T;

    for(int t = 1 ; t <= T; t++){
        int n, ans = 0;
        cin >> n;
    
        for(int i = 1; i <= n; i++){
            if(i % 2 != 0)
                ans += i;
            else
                ans -= i;
        }

        cout << '#' << t << ' ' << ans << endl;
    }
}

File: swea/D2/1986.지그재그 숫자/1986.py
T = int(input())
for tc in range(1, T+1):
    n = int(input().strip())
    
    ans = 0
    for i in range(1, n+1):
        if i % 2 == 0:
            ans -= i
    
        else:
            ans += i

    print(f"#{tc} {ans}")